--- cvs_revision_manager.py.orig	2009-08-22 17:01:09.000000000 -0500
+++ cvs_revision_manager.py	2010-08-06 23:34:31.000000000 -0500
@@ -77,8 +77,7 @@ class CVSRevisionReader(RevisionReader):
         '-r' + cvs_rev.rev,
         '-p'
         ]
-    if suppress_keyword_substitution:
-      pipe_cmd.append('-kk')
+    pipe_cmd.append('-kk')
     pipe_cmd.append(project.cvs_module + cvs_rev.cvs_path)
     return PipeStream(pipe_cmd)
 
